### PR TITLE
Replace static Cytoscape plugin with npm package

### PR DESCRIPTION
**Pull request by itisAliRH**
This PR replaces the `cytospace` visualisation static files with the npm package ([@itisalirh/galaxy-cytoscape](https://www.npmjs.com/package/@itisalirh/galaxy-cytoscape)) developed using [@guerler/galaxy-charts-starter](https://github.com/guerler/galaxy-charts-starter). Also removed duplicate <data> elements for "Depth First Search" in the XML file.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Comment by guerler** [comment]
Thank you for working on this. Can you set defaults for: `search_algorithm`, `directed` and `graph_traversal`? Also `Undirected` option has no value. These issues seem to have existed before but we can fix them now I think.

**Comment by itisAliRH** [comment]
> Thank you for working on this. Can you set defaults for: `search_algorithm`, `directed` and `graph_traversal`? Also `Undirected` option has no value. These issues seem to have existed before but we can fix them now I think.

The `directed` option is only used in the `styleGenerator` and is a boolean. I change it to a boolean input with true/false values.

The `search_algorithm` and `graph_traversal` are optional and can be used when the user selects one of them, so the default value for them can be any false values. Should I add `<value></value>` to the XML file as the default value for those?

**Comment by guerler** [comment]
Yes, I think we should add explicit values like `unset` or `null` or `default`, so users can switch back to them if necessary.

**Comment by mvdbeek** [comment]
We should publish this under the galaxyproject namespace and not individual namespaces

**Comment by guerler** [comment]
> We should publish this under the galaxyproject namespace and not individual namespaces

Why?

File: config/plugins/visualizations/cytoscape/config/cytoscape.xml [hunk 1]
```diff
@@ -13,7 +13,10 @@
     <params>
         <param type="dataset" var_name_in_template="hda" required="true">dataset_id</param>
     </params>
-    <entry_point entry_point_type="chart" src="script.js"/>
+    <requirements>
+        <requirement type="npm" version="latest" package="@itisalirh/galaxy-cytoscape"/>
```
Comment (guerler) [inline config/plugins/visualizations/cytoscape/config/cytoscape.xml]: Please specify an actual version here.

**Review by guerler — APPROVED** [review]
Package has been moved to @galaxyproject as per request of @mvdbeek. Thank you  @itisAliRH.

**Pull request merged**